### PR TITLE
Templating: Add new global built-in variables

### LINKED ISSUE #20523: Add new global built-in variables
**What would you like to be added**:
${__dashbord.name} : the dashboard name
${__org.name}: the current organization name
${__from.text}: the from variable in string format
${__to.text}: the to variable in string format

**Why is this needed**:
I need to add a title panel on each dashboard to understand on printed page what is visible.
I would like this: 
Overview : Nile (2019-11-19 13:30:40 to 2019-11-20 13:30:42)
With a Text panel with this:
`${__dashbord.name} : ${__org.name} (${__from.text} to ${__to.text})`

### PR DESCRIPTION
**Pull request by dcastanier**
What this PR does / why we need it:
It's an implementation of 'Add new global built-in variables #20523'

Which issue(s) this PR fixes:
Fixes #20523

Special notes for your reviewer:
This PR follow and replace #21690 that's had an issue with merge of previous changes

### CODE REVIEW

**Comment by dcastanier** [comment]
Now include revert change on '__from' and '__to' like requested by Dominik

**Review by dprokop — APPROVED** [review]
@dcastanier I think this looks good now, with a minor nit from @torkelo about contextSrv. Thanks for the contribution and we gonna get back with some more thoughts about formatting the from/to variables.

**Comment by marefr** [comment]
@dprokop moved this to 6.7 milestone since this is new features.

**Comment by dprokop** [comment]
thanks @marefr

File: public/app/features/templating/variable_srv.ts [hunk 1]
```diff
@@ -26,7 +27,8 @@ export class VariableSrv {
     private $location: ILocationService,
     private $injector: auto.IInjectorService,
     private templateSrv: TemplateSrv,
-    private timeSrv: TimeSrv
+    private timeSrv: TimeSrv,
+    private contextSrv: ContextSrv
```
[the remark this answers is not in the document]
Comment (dcastanier) [reply public/app/features/templating/variable_srv.ts]: got it, new commit will follow

**Review by dprokop — APPROVED** [review]

**Comment by Brombeersaft** [comment]
The docs should be updated from 6.6+ to 6.7+ as this was moved to 6.7 milestone.

**Pull request merged**